### PR TITLE
Replaced the new option in Recovering a Repository topic

### PR DESCRIPTION
**Pull request by mjivraja**
A new option was replaced i.e "Verify Content Checksum" with
Validate Content Sync as it is no longer an option for an advanced
synchronization in Satellite 6.10

"Validate Content Sync" is no longer an option for "Recovering a
Repository"

https://issues.redhat.com/browse/SATDOC-728

Cherry-pick into:

* [X] Foreman 3.2
* [X] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

### CODE REVIEW

**Review by melcorr — APPROVED** [review]

File: guides/common/modules/proc_recovering-a-corrupted-repository.adoc [hunk 1]
```diff
@@ -23,8 +23,9 @@ Use this option if you have one of the following errors:
 . In the {ProjectWebUI}, navigate to *Content* > *Products*.
 . Select the product containing the corrupted repository.
 . Select the name of a repository you want to synchronize.
-. From the *Select Action* menu, select *Advanced Sync*.
-. Select the option and click *Sync*.
+. To perform optimized sync or complete sync, select *Advanced Sync* from the *Select Action* menu.
+. Select the required option and click *Sync*.
+. To verify the checksum, click *Verify Content Checksum* from the *Select Action* menu. (optional)
```
Comment (melcorr) [inline guides/common/modules/proc_recovering-a-corrupted-repository.adoc:28]: Actually one thing I hadn't seen. This should be like this: 
```suggestion
. Optional: To verify the checksum, click *Verify Content Checksum* from the *Select Action* menu. 
```

**Review by melcorr — CHANGES_REQUESTED** [review]
One small comment. 
It doesn't matter so much for the old guide but let's get it right for the new. 
I overlooked it the last day. My apologies.

File: guides/common/modules/proc_recovering-a-corrupted-repository.adoc (hunk 1 above)
Comment (mjivraja) [reply guides/common/modules/proc_recovering-a-corrupted-repository.adoc:28]: Noted @melcorr 

I will take this minor change in the new guide i.e. v6.11 and will push it again.

Thank you.

**Review by melcorr — APPROVED** [review]

**Comment by melcorr** [comment]
Cherry picks: 
   8408d943..29db0a96  3.2 -> 3.2
   6510938d..e895cbdf  3.1 -> 3.1

**Pull request merged**